README.md

```diff
@@ -1,6 +1,102 @@
+| [SMART Health Link QR](Generic_Mobile_UX_SHL_QR.png) | QR code display screen for a particular SMART Health Link |
+| [SMART Health Link Access Log](Generic_Mobile_UX_Access_Log.png) | Access log screen for a particular SMART Health Link |
+
+## Key UX Elements
+
+### SMART Health Link Setup
+Provide users with a clear overview of the SMART Health Link they plan to create (depicted in [SMART Health Link Setup](Generic_Mobile_UX_Link_Set-up.png)). Include,
+
+* Label for the link (e.g. “Jane’s Complete Immunizations”)
+* Expiration information
+* Passcode
+* Description or list of data included in the SMART Health Link, or the action the link initiates, if not otherwise obvious in the label or UX
+
+Be sure that users understand that by activating this link, all of the referenced data will be available to anyone with access to the link.
+
+### SMART Health Link Management
+For active SMART Health Links, provide users with tools to manage the key details of that link (depicted in [SMART Health Link Management](Generic_Mobile_UX_Link_Manage.png)). Users should be able to view or configure,
+
+* The Label for the link (e.g. “Jane’s Complete Immunizations”)
+* Expiration date
+* Passcode
+* The link URI
+
+Users should also have the ability to
+
+* View the access log for the link
+* Deactivate the link
+
+### SMART Health Link Sharing
+For active SMART Health Links, provide users with tools to share that link as they see fit (depicted in [SMART Health Link Management](Generic_Mobile_UX_Link_Manage.png)).
+
+Provide users with the link URI for sharing, and make it as easy as possible to copy the link to minimize errors. This could include automating copying the URI to their clipboard or sending the URI to operating system's built-in share functionality.
+
+Make it clear to users whether or not the shared link will open a web viewer for the recipient or if the link needs to be opened by an application that understands SMART Health Links.
+
+Be sure that users understand that by sharing this link, all of the referenced data will be available to anyone with access to the link.
+
+### SMART Health Link QR Code
+For active SMART Health Links, provide users with tools to share as a QR code as they see fit (depicted in [SMART Health Link QR](Generic_Mobile_UX_SHL_QR.png)).
+
+QR codes should be presented in close proximity to the Label for the SMART Health Link so it is clear what is being shared. Include any other key information needed to identify the link and understand the contents, such as patient name or date of birth.
+
+QR Codes should be rendered in black on a white background only to maximize contrast. The QR code should have a clear white border.
+
+Embed the SMART logo in the QR code to increase recognition by users and verifiers.
```

Comment:
We currently have some language to the effect at https://docs.smarthealthit.org/smart-health-links/spec#3-data-sharer-transmits-a-shlink

Should we reference or reconcile?